ablySpec/RealtimeClientConnection.swift

```diff
@@ -1302,6 +1302,63 @@ class RealtimeClientConnection: QuickSpec {
 
             }
 
+            // RTN15
+            context("connection failures once CONNECTED") {
+
+                // RTN15a
+                pending("should be able to catch up by receiving the queued ProtocolMessages if the connection is disconnected unexpectedly") {
+                    let options = AblyTests.commonAppSetup()
+                    options.disconnectedRetryTimeout = 1.0
+
+                    let client1 = ARTRealtime(options: options)
+                    defer { client1.close() }
+                    let channel1 = client1.channels.get("test")
+
+                    let client2 = ARTRealtime(options: options)
+                    defer { client2.close() }
+                    let channel2 = client2.channels.get("test")
+
+                    var reconnected = false
+                    channel1.subscribe { message in
```

Comment:
It would be great to add one small step into here so that we can be sure the serial numbers are being respected.  Publish two messages whilst both clients are connected, then disconnect, publish a message on the connected client, and then resume the connection, and ensure the message arrives on the existing listener